dds/DCPS/security/AccessControl/Permissions.cpp

```diff
@@ -221,12 +221,14 @@ int Permissions::load(const SSL::SignedDocument& doc)
       }
     }
 
-    if (!valid_subject) {
-      ACE_DEBUG((LM_WARNING,
-        ACE_TEXT("(%P|%t) AccessControlBuiltInImpl::load_permissions_file: Unable to parse subject name, ignoring grant.\n")));
-    } else if (contains_subject_name(rule_holder_.subject)) {
-      ACE_DEBUG((LM_WARNING,
-        ACE_TEXT("(%P|%t) AccessControlBuiltInImpl::load_permissions_file: Ignoring grant with duplicate subject name.\n")));
+    if (!valid_subject && DCPS::security_debug.access_warn) {
```

Comment:
Combining these conditions means that when !valid_subject and not logging, the `contains_subject_name`  function on 228 will run.  That's a change from before.

Reply:
Yes you're right, I will fix this.